modules/ocf_desktop/files/xsession/pdf-open

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -euo pipefail
+
+[ "$#" -ne 1 ] && { echo 'Bad arguments, trying evince...'; exec evince "$@"; }
+[ -f "$1" ] || { echo 'File does not exist, trying evince...'; exec evince "$1"; }
+
+size=$(du "$1") || { echo 'Cannot get size, trying evince...'; exec evince "$1"; }
+size=$(cut -f1 <<< "$size")
+[ "$size" -lt 5120 ] || { echo 'Too large, trying evince...'; exec evince "$1"; }
+
+len=$(pdftk "$1" dump_data | awk '/NumberOfPages/{print $2}')
+[ -z "$len" ] || [ "$len" -gt 75 ] && { echo 'Too long, trying evince...'; exec evince "$1"; }
+
+filename=$(basename "$1")
+tmpdir=$(mktemp -d)
+tmp="$tmpdir/$filename"
+
+convert -density 250 "$1" "$tmp" |
+zenity --progress \
+	--pulsate \
+	--auto-close \
+	--auto-kill \
+	--text="Rasterizing File..."
+
+if [ $? -ne 0 ]; then
```

Comment:
With `set -e`, this check is superfluous because the script will bomb out if `convert` fails.

I now think that, while we should generally use `set -euo pipefail`, in this particular script the `e` part of it is counterproductive and we'd do better without it.